custom-words.txt

```diff
@@ -4,7 +4,6 @@ A256CBCPAD
 AADDS
 aadiam
 AADKERB
```

Comment:
How do you know these words to be deleted are not used in swaggers anymore?

Reply:
Also by manual review, I spotted a few 'obvious typos' like 'Seleted'.